cross-media-measurement-api/src/main/proto/wfa/measurement/api/v2alpha/measurements_service.proto

```diff
@@ -46,3 +51,42 @@ message CreateMeasurementRequest {
   Measurement measurement = 1;
 }
 
+// Request message for `ListMeasurements` method.
+message ListMeasurementsRequest {
+  // ID of the parent `MeasurementConsumer`.
+  MeasurementConsumer.Key parent = 1;
+
+  // The maximum number of `Measurement`s to return.
+  // The service may return fewer than this value.
+  // If unspecified, at most 50 `Measurement`s will be returned.
+  // The maximum value is 1000; values above 1000 will be coerced to 1000.
+  int32 page_size = 2;
+
+  // A page token, received from a previous `ListMeasurementsRequest` call.
+  // Provide this to retrieve the subsequent page.
+  //
+  // When paginating, all other parameters provided to `ListMeasurementsRequest`
+  // must match the call that provided the page token.
+  string page_token = 3;
+
+  message Filter {
+    // List of key value pair tags to retrieve `Measurement`s by.
+    // `Measurement`s with tags that have all of the tags specified here
+    // are retrieved.
+    // TODO(uakyol): extend this to cover all predicate logic.
```

Comment:
Comment says "list" when this is a map. Also drop the TODO. We can add predicate logic if and when it's needed, in which case we'll naturally update this.